sdk/search/azure-search-documents/tests/async_tests/test_service_live_async.py

```diff
@@ -197,6 +277,12 @@ async def test_analyze_text(self, api_key, endpoint, index_name, **kwargs):
         assert len(result.tokens) == 2
 
 class SearchSynonymMapsClientTest(AzureMgmtTestCase):
+    def _build_synonym_map_from_dict(self, synonym_map):
+        sm = SynonymMap(name=synonym_map["name"], synonyms=synonym_map["synonyms"])
+        for k, v in synonym_map.items():
+            setattr(sm, k, v)
+        
+        return sm
```

Comment:
this doesn't rely on `self` and is duplicated in the sync version, could be pulled into a free function in a `_test_utils` module

Reply:
fixed